src/components/obj-model.js

```diff
@@ -48,13 +48,14 @@ module.exports.Component = registerComponent('obj-model', {
     var mtlLoader = this.mtlLoader;
     var objLoader = this.objLoader;
     var rendererSystem = this.el.sceneEl.systems.renderer;
+    var basePath = mtlUrl.substr(0, mtlUrl.lastIndexOf('/') + 1);
```

Comment:
Rename to BASE_PATH. Convention used for consts

Reply:
Done 👍